libafl_bolts/Cargo.toml

```diff
@@ -95,25 +95,24 @@ rustversion = "1.0"
 libafl_derive = { version = "0.12.0", optional = true, path = "../libafl_derive" }
 static_assertions = "1.1.0"
 
-rustversion = "1.0"
```

Comment:
This is the only meaningful change here... must've gotten hit by my formatter when I updated the file.